profiles/sftp_storage/pulp_config.env

```diff
@@ -0,0 +1,7 @@
+PULP_REDIRECT_TO_OBJECT_STORAGE=false
+PULP_DEFAULT_FILE_STORAGE="pulpcore.app.models.storage.PulpSFTPStorage"
+PULP_MEDIA_ROOT=""
+PULP_SFTP_STORAGE_HOST="sftp_storage"
```

Comment:
Have you tested that this `STORAGE_HOST` is correct by performing a simple sync? I know it should be the same name of the service, but I ran into some issues when creating an S3 profile with underscores in the name, the python libraries didn't think it was a valid name and I had to switch to uses dashes.